cpp/rss_aggregator.cc

```diff
@@ -46,7 +46,7 @@ namespace {
 
 [[noreturn]] void Usage() {
     ::Usage(
-        "[--use-web-proxy] subsystem_type email_address xml_output_path\n"
+        "optional:[--download-feeds] optional:[--use-web-proxy] subsystem_type email_address xml_output_path\n"
```

Comment:
brackets `[]` already mean that something is optional, so you don't need to write `optional:` as prefix.

And since `--use-web-proxy` only makes sense if `--download-feeds` is given, the documentation should be like this:

`[--download-feeds [--use-web-proxy]]`